lldb/source/Plugins/Process/elf-core/ProcessElfCore.cpp

```diff
@@ -1095,3 +1095,9 @@ bool ProcessElfCore::GetProcessInfo(ProcessInstanceInfo &info) {
   }
   return true;
 }
+
+const uint8_t *ProcessElfCore::PeekMemory(lldb::addr_t low, lldb::addr_t high,
```

Comment:
Would an `ArrayRef<uint8_t>` be better for reading from a core file?

Reply:
I commented in the header file, ArrayRef is a good idea. We might want a `std::optional<ArrayRef<uint8_t>>` where if something with no value is returned, it means that PeekMemory is not implemented and we should fall back to the original `Process::FindInMemory`